tests/test_app.py

```diff
@@ -68,13 +68,28 @@ def test_app_configure_without_platforms(mocked_settings):
 def test_app_configure_with_tasks(mocked_engine):
     """App should have empty tasks if not defined  on engine"""
 
-    mocked_engine['instance'].tasks.return_value = []
+    mocked_engine['instance'].tasks = []
     app = App()
     app.configure_platforms()
 
     assert not app.tasks
 
 
+def test_app_configure_with_multiple_tasks(mocked_engine):
+    """App should have multiple tasks if defined  on engine"""
```

Comment:
There are two spaces between `defined` and `on engine`. Same on the test above, do you mind fixing them?

Reply:
@rougeth fixed

p.s. wow you're quick

Reply:
tks!